README.md

```diff
@@ -94,12 +94,26 @@ This section provides examples in the following frameworks:
 
 ### Java
 
+`GET` example
+
 ```
 cd ./java
 javac AWSSigner.java
 java AWSSigner
 ```
 
+`POST` example
```

Comment:
@jhart0 I don't love the fact this example falls outside the Sample App detailed above but my searches for a solution let me to this repo so I think it's the right place for it.